src/library/scala/collection/immutable/Vector.scala

```diff
@@ -888,35 +888,35 @@ private[immutable] trait VectorPointer[T] {
         display3 = copyOf(display3)
         display2 = copyOf(display2)
         display1 = copyOf(display1)
-        display5((index >> 25) & 31) = display4
-        display4((index >> 20) & 31) = display3
-        display3((index >> 15) & 31) = display2
-        display2((index >> 10) & 31) = display1
-        display1((index >>  5) & 31) = display0
+        display5((index >>> 25) & 31) = display4
```

Comment:
The error checking is in [checkRangeConvert](https://github.com/paplorinc/scala/blob/4279bbc9f467ef6d8a0ddf4190f2dd947f13d580/src/library/scala/collection/immutable/Vector.scala#L120
) and `>>>` was already used in many other places, it's the reason why I unified it: https://github.com/paplorinc/scala/blob/4279bbc9f467ef6d8a0ddf4190f2dd947f13d580/src/library/scala/collection/immutable/Vector.scala#L482